src/test/rpc_wallet_tests.cpp

```diff
@@ -1212,7 +1231,7 @@ BOOST_AUTO_TEST_CASE(rpc_wallet_encrypted_wallet_zkeys)
     strWalletPass.reserve(100);
     strWalletPass = "hello";
 
-    boost::filesystem::current_path(GetArg("-datadir","/tmp/thisshouldnothappen"));
+    PushCurrentDirectory push_dir(GetArg("-datadir","/tmp/thisshouldnothappen"));
```

Comment:
I don't think the test should be depending on the current directory at all (i.e. this is not the right fix).

Reply:
Filed a bug in https://github.com/zcash/zcash/issues/4645